### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by kojoty**
THIS IS A TEMPORARY COMMIT MADE BY CROWDIN - DON'T MERGE IT.

### CODE REVIEW

**Comment by following5** [comment]
Now all the en.php translation files at Crowdin are duplicate:

crowdin

**Comment by following5** [comment]
... and about 25% of translations are missing in the duplicate "master/lib/languages/en.php" at crowdin!

**Comment by kojoty** [comment]
I need to experiment a little with crowdin - let me do that - I need to understand how this crowdin - github integration works in details...

File: lib/languages/ro.php [hunk 1]
```diff
@@ -1322,8 +1213,8 @@
     'qrcode_appInfo' => '<a class="links" href="http://www.i-nigma.com/Downloadi-nigmaReader.html" target="_blank">AICI</a> poţi găsi aplicaţii pentru citit coduri QR pentru diferite modele de telefoane.',
     'error_ort' => 'Locating failed***',
     'alert_search_by_cachename' => 'Enter the name!***',
-    'alert_search_by_dist_01' => 'Longitudinea trebuie să fie numerică!\nFormat: GG° MM.mmm',
-    'alert_search_by_dist_02' => 'Latitudinea trebuie să fie numerică!!\nFormat: GG° MM.mmm',
+    'alert_search_by_dist_01' => 'Longitudinea trebuie să fie numerică!\\nFormat: GG° MM.mmm',
+    'alert_search_by_dist_02' => 'Latitudinea trebuie să fie numerică!!\\nFormat: GG° MM.mmm',
```
Comment (following5) [inline lib/languages/ro.php]: The additional \\ must be removed, see https://github.com/opencaching/opencaching-pl/issues/1494#issuecomment-450728859.

**Comment by kojoty** [comment]
@following5 don't worry - these changes are on separate branch - I'm not going to merge it to master for now.

**Comment by kojoty** [comment]
OK Now I understand all I wanted to know - I close this PR and crowdin branch for now

**Pull request closed without merging**